src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(Color color, double radius) {
+        super(color, Math.PI * Math.pow(radius, 2));
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create a separate variable for the area calculation and then pass it on for better code readability.